tests/unit/browser/test_mhtml.py

```diff
@@ -0,0 +1,277 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+import io
+import textwrap
+import re
+import pytest
```

Comment:
nitpick: There should be an empty line before this to group reports (builtin, libraries, projects)